kong/dao/migrations/postgres.lua

```diff
@@ -156,4 +156,43 @@ return {
       ALTER TABLE apis DROP COLUMN IF EXISTS retries;
     ]]
   },
+  {
+    name = "2016-09-16-141423_upstreams",
+    -- Note on the timestamps below; these use a precision of milliseconds
+    -- this differs from the other tables above, as they only use second precision
+    -- The Cassandra timestamps are created in Lua code, and have hence ALL at
+    -- the same time been updated to millisecond precision.
```

Comment:
If I understand correctly, this PR updates all of the existing timestamps to use `ms` precision? I am not sure if I understood this correctly. If so, where is this update performed? Except in the `db.cassandra` module where I noticed the "UNIT millisecond, and PRECISION millisecond" update?